pkgs/by-name/ut/util-linux/package.nix

```diff
@@ -27,21 +27,16 @@
 stdenv.mkDerivation rec {
   pname =
     "util-linux" + lib.optionalString (!nlsSupport && !ncursesSupport && !systemdSupport) "-minimal";
-  version = "2.39.4";
+  version = "2.40.4";
 
   src = fetchurl {
     url = "mirror://kernel/linux/utils/util-linux/v${lib.versions.majorMinor version}/util-linux-${version}.tar.xz";
-    hash = "sha256-bE+HI9r9QcOdk+y/FlCfyIwzzVvTJ3iArlodl6AU/Q4=";
+    hash = "sha256-XB2vczsE6YWa/cO9h8xIEYDuD4i1wJRrFv3skxl1+3k=";
   };
 
   patches = [
     ./rtcwake-search-PATH-for-shutdown.patch
-
-    (fetchpatch {
-      name = "basename.patch";
-      url = "https://git.kernel.org/pub/scm/utils/util-linux/util-linux.git/patch/?id=77454e58d58f904cfdc02d3ca5bb65f1bd8739fc";
-      hash = "sha256-ELWC4bYN3rvn9XIN0XgCo55pXNfS2VpbZWuwzRLfO/0=";
-    })
+    ./fix-darwin-build.patch
```

Comment:
Ideally there'd be a link to the PR so we know this has been submitted, but this has waited long enough so I'm just going to merge it.

Reply:
Added in 455db8c553d